packages/expo-notifications/android/src/main/java/expo/modules/notifications/notifications/debug/DebugLogging.kt

```diff
@@ -0,0 +1,77 @@
+package expo.modules.notifications.notifications.debug
+
+import android.os.Build
+import android.os.Bundle
+import android.util.Log
+import androidx.annotation.RequiresApi
+import com.google.firebase.messaging.RemoteMessage
+import expo.modules.notifications.BuildConfig
+import expo.modules.notifications.notifications.model.Notification
+import java.util.function.Consumer
+
+object DebugLogging {
+  @JvmStatic
+  fun logBundle(caller: String, bundleToLog: Bundle) {
+    Log.i("ReactNativeJS", "$caller:")
+    logBundle(caller, bundleToLog, "  ")
+  }
+
+  private fun logBundle(ignoredCaller: String, bundleToLog: Bundle, indent: String) {
```

Comment:
Can we change the type of the 'indent' parameter to int? Passing a string with only spaces looks really weird.

Reply:
Fixed.